testsrc/com/jambit/onboarding2020/tbrpg/games/hangman/HangmanTest.java

```diff
@@ -0,0 +1,41 @@
+package com.jambit.onboarding2020.tbrpg.games.hangman;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class HangmanTest {
+
+    private String testword = "TESTWORD";
+
+    @Test
+    void guessLetter() {
```

Comment:
There exist several naming conventions for naming test methods. Choose of it :)
https://dzone.com/articles/7-popular-unit-test-naming